src/bci_build/package/ruby/README.md.j2

```diff
@@ -0,0 +1,66 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Ruby](https://www.ruby-lang.org/) is a dynamic, reflective, object-oriented, general-purpose, open-source programming language. According to its authors, Ruby was influenced by Perl, Smalltalk, Eiffel, Ada, and Lisp. It supports multiple programming paradigms, including functional, object-oriented, and imperative. It also has a dynamic type system and automatic memory management.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application's main script:
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+WORKDIR /app
+
+COPY Gemfile Gemfile.lock ./
+RUN bundle install
+
+COPY . .
+
+CMD [ "ruby", "./main-script.rb" ]
+```
+
+Build and run the container image:
+
+```ShellSession
+$ podman build -t my-ruby-app .
+$ podman run -it --rm my-ruby-app
+```
+
+The example above expects a `Gemfile.lock` file in your application directory.
+To generate a `Gemfile.lock` file, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} bundle install
```

Comment:
```suggestion
$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} bundle lock
```

Reply:
Interesting, the official Ruby image uses `install` and not `lock`. I'll fix it.

Reply:
Adjusted.